### ISSUE TITLE
UC Team requests new property search result features (NN) SCTASK0330056 (Waiting on Granicus to decide if they want to add this to AMANDA or not)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#12465]
### New Property search results sorting features.
1. Ability to sort by folder type
2. Ability to choose records shown per page

> What application are you using?

AMANDA

> Describe the problem.

On the Folder tab of a Property module in AMANDA, the UC team wants to make the following improvements:

1. Include a Sort option that allows users to sort by Folder Type, which is not present in the current selection (see screenshot).

![Property tab.png](https://images.zenhubusercontent.com/5e8dfbfabf171441c3a5684e/3f93e8dc-4b67-4736-bc6d-de670069ffab)
 
**Steps:**
- Go to Property Menu
- Select Street Segments and search (Example; Street Number 600-615 Street Name; Davis)
- Select Property Tab
- Select Options> Sort

2. Include the ability to select how many records a user wants to be shown per page (i.e. 25, 50, 100, 200 rows per page).

> Describe the outcome you'd like to see when this feature is implemented.

The ability to sort by Folder Type will allow UC staff to more easily isolate the UC folders from the other Folder Types. The ability to select a larger number of rows per page will be helpful for UC staff on downtown streets that many contain triple digits of records.

> Describe any workarounds you currently have in place or alternative solutions you've considered.

N/A

> Requested By
Kyle K.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/647a2854288eb70026ec8e3e/sort.png) (26.08kb)

Request ID: DTS23-108085

**Comment by TracyLinder** [comment]
<img width="1032" alt="Screenshot 2023-06-02 at 2 22 33 PM" src="https://github.com/cityofaustin/atd-data-tech/assets/49930868/dce88c89-51f1-481a-af6c-81f8b23d3b90">
<img width="716" alt="Screenshot 2023-06-02 at 2 25 07 PM" src="https://github.com/cityofaustin/atd-data-tech/assets/49930868/6a9faabd-02e0-4514-8e10-d0907eaa360b">

**Comment by TracyLinder** [comment]
@Nadin-Nader please submit a SN ticket for this and move it to ongoing.

**Comment by Nadin-Nader** [comment]
@vschandramouli i added a couple of things to it but was able to send this to CTM under **SCTASK0330056**.
Moving to Ongoing Pipeline

**Comment by Nadin-Nader** [comment]
Update from Anu:

> [5:03 PM] Mahindrakar, Anu
> this one is already done and let me take a look on this card. 
> I will update on this ticket work tomorrow because it is in Vendor ticket list and Jenice is following up.

**Comment by Nadin-Nader** [comment]
Scheduled meeting for Thursday 8/10/23 @1:30pm with Jenice, Anu and Jaclyn

**Comment by Nadin-Nader** [comment]
Spoke to Jenice and she informed me that this is with the Vendor. She is investigating why they closed the ticket and why they suggested that this may not be possible to do with a foldertype but it has a Folder Subtype search

**Comment by TracyLinder** [comment]
### Convo w/ Jenice 8/22/23
- The table that populates this view doesn't include the folder type because it is joined in with the folder name. Granicus will have to decide to update their table for this to work.
- Jenice submitted a Granicus idea. 

![image](https://github.com/cityofaustin/atd-data-tech/assets/49930868/20d5f0d8-22eb-4555-ada6-52a0d65aaace)

**Comment by Nadin-Nader** [comment]
@TracyLinder since Granicus said this would take an overhaul of the table and an Idea was put in, i will be moving this to the ongoing pipeline and let Jaclyn know
